src/Uno.UWP/Media/Playback/MediaPlayer.Android.cs

```diff
@@ -161,6 +163,16 @@ protected virtual void InitializeSource()
 			}
 		}
 
+		private void SetPlaylistItems(MediaPlaybackList playlist)
+		{
+			_playlistItems = new List<Uri>();
+
+			foreach (var mediaItem in playlist.Items)
+			{
+				_playlistItems.Add(mediaItem.Source.Uri);
+			}
```

Comment:
```suggestion
			_playlistItems = playlist.Items
						.Select(i => i.Source.Uri)
						.ToList();
```

Reply:
Sure. no problem. I'm using to Azure DevOps, so we don't squash once we choose to merge the PR?